src/app/content/components/Page/PageComponent.tsx

```diff
@@ -59,6 +59,7 @@ export default class PageComponent extends Component<PagePropTypes> {
 
   public componentDidMount() {
     this.postProcess();
+    this.reApplyLocationHash();
```

Comment:
@TomWoodward I think this is another example of [what we just looked at](https://github.com/openstax/rex-web/pull/1401/files) with search highlights already being populated on mount. Is it worth looking into the root cause of this? If not I think we Bartek's solution is fine.

Reply:
the solution here seems odd to me because we have a whole thing in `scrollToTopOrHashManager` that is supposed to handle scrolling to the hash. if the real issue is about the props not updating as you addressed in your other PR then maybe we need to also add a call to it's update here like you did for search highlights